sites/public/src/pages/sign-in.tsx

```diff
@@ -17,8 +17,10 @@ import {
   ResendConfirmationModal,
   FormSignInDefault,
   FormSignInPwdless,
+  NetworkErrorMessage,
 } from "@bloom-housing/shared-helpers"
 import { UserStatus } from "../lib/constants"
+import PasswordExpiredModal from "../components/account/PasswordExpiredModal"
```

Comment:
issue: vscode warns of an import format error here 
```
Using exported name 'PasswordExpiredModal' as identifier for default import
```
It looks like the PasswordExpiredModal.tsx is exporting differently at the bottom of the file compared to our other components